voluncheer/templates/profiles/profile.html

```diff
@@ -287,27 +283,30 @@ <h2>PAST OPPORTUNITIES</h2>
         </div>
 
         <div class="sidenav-url">
+            {% if user.pk is curr_user.pk %}
```

Comment:
PROFILE shouldn't be restrained from others

Reply:
If it's a different user, then they wouldn't be able to click other user's profile right? Right now, it redirects to the user's own profile when they click on other's profile.